app/assets/stylesheets/content/_tooltips.sass

```diff
@@ -26,27 +26,160 @@
 // See doc/COPYRIGHT.rdoc for more details.
 //++
 
-/***** Tooltips *****
+//***** Tooltips *****
 
-.tooltip
+// tooltips used for arbitrary elements
+
+$tooltipBackground: #e3f5ff
+$tooltipBorder: none
+$tooltipHeight: 22px
+$tooltipTextColor: $body-font-color
+$tooltipArrowSize: 6px
+
+%top-bottom
+  &:before, &:after
+    left: 50%
+    transform: translateX(-50%)
+
+  &:after
+    width: auto
+
+  &.-multiline
+    &:before, &:after
+      transform: translateY(50%)
+      margin-bottom: 0
+      filter: blur(10px)
+
+%left-right
+  &:before, &:after
+    bottom: 50%
+  &:before
+    margin-bottom: $tooltipArrowSize * -1 + 1
+  &:after
+    margin-bottom: $tooltipHeight/-1.5
+
+  &.-multiline
+    &:before, &:after
+      transform: translateY(50%)
+      margin-bottom: 0
+
+@mixin colorize($color:$tooltipBackground, $textColor:$tooltipTextColor)
```

Comment:
@floriank similar to above comment: https://github.com/opf/openproject/pull/2795/files#r27984246